ydb/src/load_balancer.rs

```diff
@@ -182,6 +215,247 @@ pub(crate) async fn update_load_balancer(
     }
 }
 
+pub(crate) struct BalancerConfig {
+    fallback_strategy: FallbackStrategy,
+}
+
+// What will balancer do if there is no available endpoints at local dc
+pub(crate) enum FallbackStrategy {
```

Comment:
What about use other balancer as fallback?

Or use two balancers: for balancing nodes in nearest DC and fallback balancer.